Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:10
+
+# Change working directory
+WORKDIR "/telescope"
+
+# Update packages and install dependency packages for services
+RUN apt-get update \
+ && apt-get dist-upgrade -y \
+ && apt-get clean \
+ && echo 'Finished installing dependencies'
+
+# Copy package.json and package-lock.json
+COPY package*.json ./
+
+# Install npm production packages 
+RUN npm install --production
+
+COPY . .
+
+ENV NODE_ENV production
+ENV PORT 3000
```

Comment:
These should both work with our `.env`, and override via environment variables, but we should confirm, since we expect to be able to set them via `.env`